modules/airgap/setup/bastion_prepare.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+## Uncomment the following lines to enable debug mode
+#set -x
```

Comment:
set -x is going to be commented or already available as line 7 ?

Reply:
Will be commented

Reply:
Updated